packages/components/src/Tabs/_docs/Tabs.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { Button } from "~components/Button"
+import { Card } from "~components/Card"
+import { Text } from "~components/Text"
+import { Tabs, TabList, Tab, TabPanels, TabPanel } from "../index"
+
+const meta = {
+  title: "Components/Tabs",
+  component: Tabs,
+  args: {},
```

Comment:
You can add the `children` here which should apply them to all stories

Reply:
Nice good call